client/state/action-types.ts

```diff
@@ -624,6 +624,9 @@ export const PLUGINS_RECEIVE = 'PLUGINS_RECEIVE';
 export const PLUGINS_REQUEST = 'PLUGINS_REQUEST';
 export const PLUGINS_REQUEST_FAILURE = 'PLUGINS_REQUEST_FAILURE';
 export const PLUGINS_REQUEST_SUCCESS = 'PLUGINS_REQUEST_SUCCESS';
+export const PLUGINS_ALL_REQUEST = 'PLUGINS_ALL_REQUEST';
```

Comment:
Do you mind sorting these so all action types continue to be in alphabetical order?

Reply:
Thanks! Fixed in 507aa423b5f3e310d59aee2b4c8494ac112cd791